package.json

```diff
@@ -4,9 +4,17 @@
   "description": "<a name=\"readme-top\">Organize your time with this simple application, which will allow you to manage your own to-do list.</a>",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack --mode=development"
```

Comment:
- Currently, you do not have a start script in your project, I'll suggest you add one as it is used to run the project while developing instead of having to build your js file before viewing it in the browsers. :wave:  You can add this to your `scripts`: `"start": "webpack serve --open"`, you also need to install the `webpack-dev-server` :smile: :small_airplane: